lucene/core/src/java/org/apache/lucene/index/ConcurrentMergeScheduler.java

```diff
@@ -516,18 +519,18 @@ public synchronized void merge(IndexWriter writer, MergeTrigger trigger) throws
 
     if (verbose()) {
       message("now merge");
-      message("  index: " + writer.segString());
+      message("  index(source): " + mergeSource.toString());
```

Comment:
did you mean to add `(source)` after `index` or is it a side-effect of a search/replace?

Reply:
I added it on purpose.